app/assets/javascripts/snowcrash/modules/notifications.js.coffee

```diff
@@ -0,0 +1,21 @@
+document.addEventListener 'turbolinks:load', ->
+  if $('.js-notifications-dropdown').length
+    $container = $('.js-notification-container')
+    $count     = $('.js-notification-count')
```

Comment:
Why not render the whole dropdown serverside?
We could skip almost all this js.